src/path.rs

```diff
@@ -0,0 +1,64 @@
+use crate::{COMPANY_NAME, COMPANY_TLD, REPOSITORY_URL};
+use directories::ProjectDirs;
+use std::path::PathBuf;
+
+/// Gets a cache directory, based on operation system
```

Comment:
Pls add this info to README.md to be clear where we store packages.

Reply:
Fixed